tests/content-routes.js

```diff
@@ -0,0 +1,153 @@
+import test from 'ava';
+import events from 'events';
+import httpMocks from 'node-mocks-http';
+import _ from 'lodash';
+
+import routes from '../lib/content/routes';
+import typesMock from './fixtures/types-post-merge';
+import allFlows from './fixtures/workflows/all-flows';
+
+const references = [
+  {
+    type: 'test-reference',
+    attr: 0,
+    input: 'reference',
+    ct: 'test-service',
+  },
+  {
+    type: 'test-reference',
+    attr: 1,
+    input: 'reference',
+    length: 1,
+    ct: 'test-service',
+  },
+];
+
+const EventEmitter = events.EventEmitter;
+
+//////////////////////////////
+// Routes - All Content Types landing
+//////////////////////////////
+test.cb('All content home route', t => {
+  const response = httpMocks.createResponse({ eventEmitter: EventEmitter });
+  routes.all(response, typesMock);
+
+  response.on('end', () => {
+    const data = response._getRenderData();
+
+    t.is(response.statusCode, 200, 'Should be a 200 response');
+    t.is(data.content.home.title, 'All Content Types');
+    t.is(data.content.base, 'content');
+    t.is(data.content.types, typesMock);
+    t.end();
+  });
+  response.render();
+});
+
+//////////////////////////////
+// Routes - One content type's landing route
+//////////////////////////////
+test.cb('Working single content landing route', t => {
+  const response = httpMocks.createResponse({ eventEmitter: EventEmitter });
```

Comment:
Is `node-mocks-http` a replacement for supertest? If so, why are we using this instead of supertest?